shell/platform/embedder/embedder.h

```diff
@@ -592,6 +592,28 @@ FlutterEngineResult FlutterEngineDispatchSemanticsAction(
     const uint8_t* data,
     size_t data_length);
 
+// A profiling utility. Logs a trace duration begin event to the timeline. If
+// the timeline is not available or disabled, this has no effect. Must be
```

Comment:
Change "not available" to "unavailable" in these comments to reduce ambiguity

Reply:
Done.